deftree/deftree.go

```diff
@@ -136,23 +136,36 @@ func (self *MicroserviceDefinition) GetByName(name string) Describable {
 	return nil
 }
 
+type notFoundErr struct {
```

Comment:
This seems like it doesn't need to be a struct.
```
type notFoundErr error
```
would be sufficient, and it can be instantiated like
```
err := notFoundErr(fmt.Errorf("..."))
```